runtime/commonMain/src/kotlinx/serialization/SerializerResolving.kt

```diff
@@ -80,3 +85,6 @@ public fun serializer(type: KType): KSerializer<Any?> {
     val result = serializerByKTypeImpl(type)
     return if (type.isMarkedNullable) result.nullable else result as KSerializer<Any?>
 }
+
+@ImplicitReflectionSerializer
+internal expect fun <T : Any> KClass<T>.invokeSerializerGetter(vararg args: KSerializer<Any?>): KSerializer<T>?
```

Comment:
Please document this method and rename it to something more appropriate (currently, it's just an exposure of JVM implementation details).
For example, `lookupSerializerInCompanion`

Reply:
I'm not sure if documentation for `internal` method is really required